tests/unit-tests/test-class-usage-tracking.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+class Sensei_Usage_Tracking_Test extends WP_UnitTestCase {
+	public function setup() {
+		parent::setup();
+		$this->factory = new Sensei_Factory();
+		$this->usage_tracking = new Sensei_Usage_Tracking();
+	}
+
+	public function teardown() {
+		parent::teardown();
+	}
+
+	/**
+	 * Ensure cron job action is set up.
+	 */
+	public function test_cron_job_action_added() {
```

Comment:
You tend to see camel case being used for PHPUnit function names. Since it's the style that the rest of Sensei's tests use as well, probably better to use camel case.

Also, you can use the `@covers` annotation to show which functions are being tested as part of this test case, since it's unclear from the function name and I'd need to look inside the function to figure it out. I usually try to do both (`@covers` + function name).

BTW, this test is going to need to be affected by https://github.com/Automattic/sensei/pull/1983. 😬

Reply:
Switched to camel case (I was debating this, and didn't notice that the rest of Sensei used camel case for tests. Good catch!)

Added `@covers` annotation.

Re: #1983 now that it's merged, I'll rebase this and update accordingly.